### DISCUSSION TITLE
Install new release on ubuntu

### DISCUSSION THREAD

**Discussion by dmtzs** [bitnami-labs/sealed-secrets#908] [General]
I was trying to install kubeseal in ubuntu and I use the command below:
```
wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.16.0/kubeseal-linux-amd64 -O kubeseal
```
However this is not the recently release, I tried to use the same command above changing the release version without success, is there a way to install the recently one? Or Im missing something?

**Comment by mkmik** [comment]
The URL filename changed a bit

https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.18.1/kubeseal-0.18.1-linux-amd64.tar.gz
